dime-research-standards/pillar-3-research-reproducibility/checklists/tex/reproducibility-package-guidance-note.tex

```diff
@@ -0,0 +1,109 @@
+DIME Analytics follows specific tasks when conducting the computational reproducibility checks, which are outlined below.
+
+	\subsection{1. \, Receving the reproducibility package}
+
+	The project team will send a code review request via email to DIME Analytics by providing a reproducibility package in a shared OneDrive or Dropbox folder, a GitHub repository, or a compressed folder sent by email if the files size allows. If using a GitHub repository, please share only the code files through GitHub and send the input data files using another channel. The package should contain the items listed below.
+
+	\bigskip
+
+	\begin{itemize}
+		\setlength\itemsep{-0.1em}
+		\item \textbf{Code:} All scripts necessary to reproduce the analytical outputs of the paper, including a well-documented and well-commented Master Script that allows to perform a one-button run of the code.
+		\item \textbf{Data:} Fully de-identified, no access-restricted, and complete data necessary to run the master script that reproduces the outputs (can be raw, clean, or constructed) . Any exception where access to restricted or identified datasets is needed to reproduce code outputs will be handled on a case-by-case basis.
+		\item \textbf{Output:} All raw outputs used for the paper (tables and figures), as created by the master script.
+	\end{itemize}
+
+	\bigskip
+
+	Note: All the items listed above should be stored in different subfolders in the main project directory.
+
+	\bigskip
+
+	\begin{itemize}
+		\setlength\itemsep{-0.1em}
+		\item \textbf{Working Paper:} Paper to compare the output generated by the code with the exhibits as presented in the paper.
+		\item \textbf{README.txt:} Text file delineating the requirements of the code and datasets included in the package, instructions about the directory structure DIME Analytics should follow, and any ad-hoc instructions to run the master script.
+		\item \textbf{Filled-in reproducibility check checklist:} DIME Analytics ask submitting teams to provide \href{https://raw.githubusercontent.com/worldbank/dime-standards/master/dime-research-standards/pillar-3-research-reproducibility/checklists/Reproducibility%20package%20submission%20checklist.pdf}{this checklist} so they can make sure they are submitting all the files needed to run a reproducibility package.
+	\end{itemize}
+
+	DIME Analytics will only change the topmost global directory specified in the master script and will run it to reproduce the results – this is what is called a one-button run. The master script code should run without errors after adding the correct folder path of the computer where it will be running. For the paper to be considered fully reproducible, DIME Analytics should reproduce the every exhibit in the paper showing analysis results through this push-button exercise.
```

Comment:
it's after. But now that you're mentioning it, perhaps we should start doing it at this stage. It would help to catch early if the code is not complete. It doesn't make sense to go on with the full review if the code is incomplete

Reply:
I added this to the one-button run in 284653e